book/website/reproducible-research/renv/renv-containers.md

```diff
@@ -420,6 +430,120 @@ docker run -d --name dind-rootless --privileged docker:20.10-dind-rootless
 
 The following prequisites, which are part of the [`shadow-utils`](https://github.com/shadow-maint/shadow) package are required to run Docker rootless: `newuidmap` and `newgidmap`.
 
+(rr-renv-containers-podman)=
+## Podman
+
+[Podman](https://podman.io/) is an open-source container engine that can be seen as a compelling alternative to Docker.
```

Comment:
Just flagging that we do say "lightly" opinionated guides, so perhaps another, lighter adjective than "compelling" to be used here ;)

Reply:
Any suggestions? Meaningful?

I think I want to emphasise that there may be strong reasons for choosing Podman over Docker despite the interface being very similar.

Reply:
Fixed in [4e773c0](https://github.com/alan-turing-institute/the-turing-way/pull/2370/commits/4e773c0533d6363bdfd336054ce3181d83c59346)